tests/default.nix

```diff
@@ -110,6 +110,7 @@ in import nmtSrc {
     ./modules/programs/ncmpcpp
     ./modules/programs/ne
     ./modules/programs/neomutt
+    ./modules/programs/jqp
```

Comment:
you should probably sort it by `alphabetic` order

Reply:
Sure, but there is the issue that created test are not passing. Will it help?

Reply:
I do not think so

I sadly can't help. I am a new comer to `nix` and `home-manager`, and started learning by checking the `PR` and found yours

I hope someone with a proper experience will be able to help (as I would also love to learn why it is going wrong)